ddl/backfilling_read_index.go

```diff
@@ -144,6 +144,10 @@ func (r *readIndexExecutor) RunSubtask(ctx context.Context, subtask *proto.Subta
 		return err
 	}
 
+	// TODO(tangenta): when we support running subtasks concurrently,
+	// we should ignore "concurrency quota exceeded" error if possible.
+	r.bc.ResetWorkers(r.job.ID, r.index.ID)
```

Comment:
Do other executor need reset workers?

Reply:
No, you can search `BackendCtx.Register`, there are only two usages:

1. `ingestBackfillScheduler.createWorker()`, we already reset the worker count when the import job complete.
2. `readIndexExecutor.RunSubtask()` here.